readme.md

```diff
@@ -59,7 +61,7 @@ gulp.task('default', function () {
 ## Babel Metadata
 
 Files in the stream are annotated with a `babel` property, which
-contains the [metadata](http://babeljs.io/docs/advanced/external-helpers/#selective-builds) from `babel.transform()`.
```

Comment:
We should link to the Babel docs on this one, otherwise the user has no idea what it will contain.